framework/midi/miditypes.h

```diff
@@ -65,30 +65,23 @@ struct SynthesizerState {
         Val() = default;
         Val(ValID id, const std::string& val)
             : id(id), val(val) {}
+
+        bool operator ==(const Val& other) const { return other.id == id && other.val == val; }
+        bool operator !=(const Val& v) const { return !operator ==(v); }

```

Comment:
Better to name parameter as other like in == method.

Reply:
fixed